### ISSUE TITLE
Heltec Automation Question

### ISSUE THREAD

**Issue by BSMantaray** [helium/miner#1813]
The Heltec miners in my area have frozen out all other miners.  Heltec will only respond to Heltec beacons.  Is this by design?

**Comment by kudabelangfxrace** [comment]
My area full of haltec automation spoofer. They are only beacon and witness within their spoofer machine. it frozen out my miner. No receive income.

**Comment by kudabelangfxrace** [comment]
![Uploading 1825F558-9FA4-435F-B944-E3589BDF13A5.png…]()

**Comment by kudabelangfxrace** [comment]
Rural Garnet Whale
Tiny Mustard Barbel

All around this location are haltec automation miner spoofer

**Comment by BSMantaray** [comment]
I am not certain if they are spoofers but these Heltec miners have
definitely frozen me out as well, and it looks like they only communicate
with each other:

Spicy Fossilized Shell
Pet Parchment Owl
Sticky Pastel Haddock
Handsome Felt Snail
Rapid Seaweed Sidewinder
Helpful Inky SeagullJumpy Boysenberry Puppy

I am not too clear on the tech they are using as their range on reported
dBi doesn't make sense.  They are getting crazy range on 1.2 dBi.

**Comment by kudabelangfxrace** [comment]
Yes, they seems using a standard antenna. 1.2 dbi. Yes they only communicate with each of them. But get our beacon yo get their reward without witness us ( honest miner). . 

Ive been told to add them heltec miner to deny list 

https://github.com/helium/denylist


List them and kick out..

**Comment by kudabelangfxrace** [comment]
Bunch of haltec miner has been reported if you see in the issue list

**Comment by BSMantaray** [comment]
Thank you for the information.  I will submit to the denylist.
